sim/firesim-lib/src/main/scala/bridges/GroundTestBridge.scala

```diff
@@ -0,0 +1,41 @@
+//See LICENSE for license details
+package firesim.bridges
+
+import midas.widgets._
+
+import chisel3._
+import chisel3.util._
+import freechips.rocketchip.config.Parameters
+
+class GroundTestBridge extends BlackBox
+    with Bridge[HostPortIO[GroundTestBridgeTargetIO], GroundTestBridgeModule] {
+  val io = IO(new GroundTestBridgeTargetIO)
+  val bridgeIO = HostPort(io)
+  val constructorArg = None
+  generateAnnotations()
+}
+
+object GroundTestBridge {
+  def apply(success: Bool)(implicit p: Parameters): GroundTestBridge = {
+    val bridge = Module(new GroundTestBridge)
+    bridge.io.success := success
+    bridge
+  }
+}
+
+class GroundTestBridgeTargetIO extends Bundle {
+  val success = Input(Bool())
+}
+
+class GroundTestBridgeModule(implicit p: Parameters)
+    extends BridgeModule[HostPortIO[GroundTestBridgeTargetIO]] {
+  val io = IO(new WidgetIO)
+  val hPort = IO(HostPort(new GroundTestBridgeTargetIO))
+
+  val tFire = hPort.toHost.hValid && hPort.fromHost.hReady
+  hPort.toHost.hReady := tFire
+  hPort.fromHost.hValid := tFire
```

Comment:
Tie both of these off to true.B

Reply:
You don't produce output tokens. We need avoid this pattern of making the channels dependent on one another whenever possible.